engine/gui/src/gui.cpp

```diff
@@ -4229,10 +4231,15 @@ namespace dmGui
         InternalNode* out_n = &scene->m_Nodes[index];
         memset(out_n, 0, sizeof(InternalNode));
 
+        // generate a name for the cloned node
+        char name[16];
+        sprintf(name, "node%d", g_ClonedNodeCount++);
```

Comment:
One question is if we perhaps should chose node ids that are more uncommon than this?

* clonednode%d
* _node%d
* _clonednode%d

Reply:
That was my first reaction, that this is easy to accidentally reproduce.
I agree, this needs to be more robust.

Reply:
Also, use dmSnPrintf() which is cross platform

Reply:
> I agree, this needs to be more robust.

Does `__node%d` is robust enough? Or perhaps better to add the number first? Something like `_%d_node`